spec/models/page_spec.rb

```diff
@@ -385,6 +385,37 @@ module Alchemy
       end
     end
 
+    describe '.all_locked_by' do
+      let(:user) { double(:user, id: 1, class: DummyUser) }
+
+      before do
+        FactoryGirl.create(:public_page, :locked => true, locked_by: 53) # This page must not be part of the collection
```

Comment:
Mixed Hash key syntax. Please use either 1.8 syntax or 1.9, but please don't mix them in one Hash, thanks.

Same on L399